bim2sim/task/common/common.py

```diff
@@ -589,6 +591,24 @@ def run(self, workflow: Workflow, ifc) -> [dict, dict]:
         self._write_errors_to_html_table(self.plugin)
         return [self.error_summary_sub_inst, self.error_summary_inst],
 
+    @staticmethod
+    def check_ifc_version(ifc: file):
```

Comment:
Looks reasonable, but is this already executed within the CheckIfcBPS class? This function is not entered in my case, even if I add CheckIfcBPS to my workflow.

Reply:
Short answer: yes

Long answer: `CheckIfcBPS` class inherits from `CheckIfc` class and doesn't have it's own `run()` method. This way, the `run()` method of the `CheckIfcBPS` class is always called, independent which check you are running. And the `run()` method of `CheckIfc` calls `self.check_ifc_version(ifc)`.